packages/nuxt-windicss/package.json

```diff
@@ -30,7 +27,7 @@
   },
   "files": [
     "dist",
-    "CHANGELOG.md"
```

Comment:
Tip: `CHANGELOG.md`, `README.md` and `LICENSE.md` are by default added to package even if not listed in `files`